### ISSUE TITLE
No indication of pressed or not when theme is "Material"

### ISSUE THREAD

**Issue by brainz80** [HeidiSQL/HeidiSQL#1976]
**Preconditions**

* HeidiSQL version: 12.7.0.6906 (64 bit)
* Database type and version: MySQL 5.7.33-0ubuntu0.16.04.1-log
* OS: Windows 11 (OS Build 22631.3593)

**Describe the bug**

There's no indication if button (e.g. Stop on errors in batch mode) is pressed or not.

**To Reproduce**

Switch theme to "Material" and check the buttons / icons.

**Screenshots or Crash reports**

## Material:
![image](https://github.com/user-attachments/assets/95187001-7c33-45bf-99c9-484b1747e372)

## Glow:
![image](https://github.com/user-attachments/assets/09f12cb3-103b-4c5d-b235-c3354e2320a5)

## Light:
![image](https://github.com/user-attachments/assets/59e294e9-419b-4407-9908-9f535a41c23f)

**Comment by ansgarbecker** [comment]
Yes, the Material theme has quite some more of these contrast killers. v13 will have a different theming - just dark or light, and no bitmap themes. Here's two screenies of how that pressed button looks in that codebase on light and dark (on a 100% DPI screen). I think that should be ok.

light:
<img width="206" height="36" alt="Image" src="https://github.com/user-attachments/assets/6cfedf73-0128-4b27-b6f4-f9179706b443" />

dark:
<img width="208" height="37" alt="Image" src="https://github.com/user-attachments/assets/c0a00707-992d-451c-9a0a-11fcd4654d3b" />

Here's a full screenshot: https://www.heidisql.com/screenshots.php?which=dark-mode-v13

You may also test the v13 preview release from [the download page](https://www.heidisql.com/screenshots.php?which=dark-mode-v13). Just unpack it into your existing HeidiSQL folder.
